components/common/AppReportModal.vue

```diff
@@ -34,7 +34,7 @@
           <app-button
             class="h-full w-full"
             text-class="small-text"
-            :text="$t(btnCopyEmailTitle)"
+            :text="btnCopyEmailTitle"
```

Comment:
nitpick(non-blocking): instead of splitting the enum into variables (line 150 & 152) you should reference it directly, `EmailEnum.Address` here and for places with the Pgp `EmailEnum.Pgp`